src/sparseml/yolov8/validators.py

```diff
@@ -36,7 +36,13 @@ def __call__(self, trainer=None, model=None):
         if self.training:
             self.device = trainer.device
             self.data = trainer.data
-            model = trainer.ema.ema or trainer.model
+            if trainer.manager.quantization_modifiers:
+                # Since we disable the EMA model for QAT, we validate the non-averaged
+                # QAT model
+                model = de_parallel(trainer.model)
```

Comment:
👍 this was the original reason i thought to just always use trainer.model, but implemented more generally!